homeassistant/components/shelly/strings.json

```diff
@@ -21,7 +21,8 @@
     "error": {
       "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
       "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
-      "unknown": "[%key:common::config_flow::error::unknown%]"
+      "unknown": "[%key:common::config_flow::error::unknown%]",
+      "firmware_not_fully_provisioned": "Device not fully provisioned. Please contact Shelly support"
```

Comment:
```suggestion
      "firmware_not_fully_provisioned": "[%key:common::config_flow::error::firmware_not_fully_provisioned%]"
```

Reply:
I tested this on a live system, without this change the form display an empty box